src/tsickle.ts

```diff
@@ -1760,12 +1765,13 @@ class ExternsWriter extends ClosureRewriter {
     }
   }
 
-  private writeExternsVariableDecl(decl: ts.VariableDeclaration, namespace: string[]) {
+  private writeExternsVariableDecl(
+      decl: ts.VariableDeclaration, namespace: string[], extraTags: jsdoc.Tag[] = []) {
     if (decl.name.kind === ts.SyntaxKind.Identifier) {
       const name = getIdentifierText(decl.name as ts.Identifier);
       if (closureExternsBlacklist.indexOf(name) >= 0) return;
-      this.emitJSDocType(decl);
-      this.emit('\n');
+      const type = this.typeToClosure(decl);
```

Comment:
The reason emitJSDocType emits whitespace first is that it is replacing the existing trivia on the node.  If you had input like `foo\n\nbar` and we add jsdoc to `bar`, we need some whitespace in place of the `\n`.  I think in the case of externs we are emitting a line at a time (and jsdoc.toString inserts a newline) so your new layout is correct.  In retrospect we probably should've attempted to replicate the whitespace of whatever content we're deleting, but at the time we wrote this we didn't think the output needed to be very human-readable.